stdlib/universe/join.go

```diff
@@ -144,13 +142,6 @@ func createJoinOpSpec(args flux.Arguments, a *flux.Administration) (flux.Operati
 	return spec, nil
 }
 
-func (t *JoinOpSpec) IDer(ider flux.IDer) {
-	for i, name := range t.params.names {
-		operation := t.params.operations[i]
-		t.TableNames[ider.ID(operation)] = name
-	}
-}
```

Comment:
Implementing this method is no longer needed now that we aren't using the operation ids to figuratively _shoot ourselves in the foot_.